tests/test_md_cli.py

```diff
@@ -69,54 +69,72 @@ def test_md(ensemble, tmp_path):
     assert isinstance(atoms, Atoms)
 
 
-def test_log(tmp_path):
+def test_log():
     """Test log correctly written for MD."""
-    file_prefix = tmp_path / "nvt-T300"
-    log_path = tmp_path / "test.log"
-    summary_path = tmp_path / "summary.yml"
-
-    result = runner.invoke(
-        app,
-        [
-            "md",
-            "--ensemble",
-            "nvt",
-            "--struct",
-            DATA_PATH / "NaCl.cif",
-            "--file-prefix",
-            file_prefix,
-            "--steps",
-            20,
-            "--stats-every",
-            1,
-            "--log",
-            log_path,
-            "--summary",
-            summary_path,
-        ],
-    )
-    assert result.exit_code == 0
-
-    assert_log_contains(log_path, includes=["Starting molecular dynamics simulation"])
-
-    with open(tmp_path / "nvt-T300-stats.dat", encoding="utf8") as stats_file:
-        lines = stats_file.readlines()
-        # Includes step 0
-        assert len(lines) == 22
-
-        # Test constant volume
-        assert lines[0].split(" | ")[8] == "Volume [A^3]"
-        init_volume = float(lines[1].split()[8])
-        final_volume = float(lines[-1].split()[8])
-        assert init_volume == 179.406144
-        assert init_volume == pytest.approx(final_volume)
-
-        # Test constant temperature
-        assert lines[0].split(" | ")[16] == "Target T [K]\n"
-        init_temp = float(lines[1].split()[16])
-        final_temp = float(lines[-1].split()[16])
-        assert init_temp == 300.0
-        assert final_temp == pytest.approx(final_temp)
+    final_path = Path("./NaCl-final.extxyz").absolute()
+    stats_path = Path("./NaCl-stats.dat").absolute()
+    traj_path = Path("./NaCl-traj.extxyz").absolute()
+    log_path = Path("./NaCl-log.yml").absolute()
+    summary_path = Path("./NaCl-summary.yml").absolute()
```

Comment:
Are we also testing the cases when the ensemble parameters and temperatures get added in to the log and summary?

Maybe also appropriate in ```tests/test_md.py```.

Reply:
These will change once #279 is fixed